### PR TITLE
[CDA-119] Scriptable JavaScript Data Source

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pamval please review

### CODE REVIEW

**Comment by carlosrusso** [comment]
Yay! My dream datasource!

**Comment by pamval** [comment]
CDA samples must be extended with an example of this datasource.

Is this appearing correctly in CDE ?

Is there an example of the query in the query field when the user creates a new query (like the MQL or the Scriptable ?)

**Comment by davidmsantos90** [comment]
Created CDA samples for this datasource, it is appearing correctly in CDE and made a push to CDE to also include an example query when using the new datasource

@pamval please review

File: cda-core/test-resources/repository/sample-json-scripting.cda [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ["2006 Q2", 410],
+    ["2006 Q3", 340],
+    ["2006 Q4", 353]
+  ],
+  "metadata": [
+    { "colIndex": 0, "colType": "String", "colName": "Year-Quarter" },
+    { "colIndex": 1, "colType": "Numeric", "colName": "Value" }
+  ]
+}
+return model;
```
Comment (carlosrusso) [inline cda-core/test-resources/repository/sample-json-scripting.cda]: Humm,  what's this?

**Comment by davidmsantos90** [comment]
Removed that extra line in sample-json-scripting.cda, wasn't there doing anything
@pamval please review

**Pull request merged**